to-rdjson.jq

```diff
@@ -0,0 +1,31 @@
+# Convert hadlint JSON output to Reviewdog Diagnostic Format (rdjson)
+# https://github.com/reviewdog/reviewdog/blob/f577bd4b56e5973796eb375b4205e89bce214bd9/proto/rdf/reviewdog.proto
+{
+  source: {
+    name: "hadolint",
+    url: "https://github.com/hadolint/hadolint"
+  },
+  diagnostics: . | map({
+    message: .message,
+    code: {
+      value: .code,
+      url: (if .code | startswith("DL") then
+              "https://github.com/hadolint/hadolint/wiki/\(.code)"
+            elif .code | startswith("SC") then
+              "https://github.com/koalaman/shellcheck/wiki/\(.code)"
```

Comment:
I think it came from another linter 🙂

Reply:
Yes... and it's expected. See https://github.com/hadolint/hadolint

Reply:
I got it. Thank you.